lib/apiservers/engine/backends/archive.go

```diff
@@ -146,7 +146,14 @@ func (c *Container) importToContainer(op trace.Operation, vc *viccontainer.VicCo
 	mounts := mountsFromContainer(vc)
 	mounts = append(mounts, types.MountPoint{Destination: "/"})
 	writerMap := NewArchiveStreamWriterMap(op, mounts, target)
-	defer writerMap.Close() // This should shutdown all the stream connections to the portlayer.
+	defer func() {
+		// This should shutdown all the stream connections to the portlayer.
+		e1 := writerMap.Close()
+		if err == nil {
```

Comment:
This is to get errors from portlayer calls. I've updated Close() to wait for all portlayer calls to return and return errors from portlayer if any.